vnext/Shared/Modules/BlobModule.h

```diff
@@ -11,6 +11,9 @@
 // React Native
 #include <cxxreact/CxxModule.h>
 
+// Boost Libraries
+#include <boost/uuid/uuid_generators.hpp>
```

Comment:
Would it be possible to wrap up the GUID generation into a special utility class? It would be nice to avoid using boost headers from our header files. Boost is too hungry for resources.

Reply:
Sure can.

That said, this header is only meant to be included once per DLL, so it should not have a significant impact as is

I'll still do the utility class in case we need guid generation elsewhere in the future.

Reply:
I have decided to make this functionality shared in a subsequent PR.
This implementation requires object lifetime management so can't be trivially exposed as a flat function.